benchmark_runner/main/environment_variables.py

```diff
@@ -45,6 +45,19 @@ def __init__(self):
                                                          'hammerdb_pod_postgres', 'hammerdb_vm_postgres', 'hammerdb_kata_postgres',
                                                          'hammerdb_pod_mssql', 'hammerdb_vm_mssql', 'hammerdb_kata_mssql',
                                                          'vdbench_pod', 'vdbench_kata']
+        # benchmark-operator workload types
+        self._environment_variables_dict['workload_types'] = {
+            'stressng': 'benchmark_operator',
+            'hammerdb': 'benchmark_operator',
+            'uperf': 'benchmark_operator',
+            'vdbench': 'benchmark_runner',
+            }
```

Comment:
I think the name should ['workload_namespace']

Reply:
Done.